plugins/GeoIp2/GeoIP2AutoUpdater.php

```diff
@@ -186,8 +200,10 @@ public static function unzipDownloadedFile($path, $dbType, $unlink = false)
             $fileToExtract = null;
             foreach ($content as $info) {
                 $archivedPath = $info['filename'];
-                if (in_array(basename($archivedPath), LocationProviderGeoIp2::$dbNames[$dbType])) {
-                    $fileToExtract = $archivedPath;
+                foreach (LocationProviderGeoIp2::$dbNames[$dbType] as $dbName) {
+                    if (basename($archivedPath) === $dbName || preg_match($dbName, basename($archivedPath))) {
```

Comment:
Running the updater this throws a warning:
```
WARNING [2019-12-29 20:09:57] 31960  /srv/matomo/plugins/GeoIp2/GeoIP2AutoUpdater.php(204): Warning - preg_match(): Delimiter must not be alphanumeric or backslash
```

Reply:
Should be fixed